modules/microshift-config-snippets.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-yaml.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-config-snippets_{context}"]
+= Using configuration snippets
+
+If you want to configure one or two settings, such as adding subject alternative names (SANs), you can use the  `/etc/microshift/config.d/` configuration directory to drop in configuration snippet YAML files. You must restart {microshift-short} for new configurations to apply.
+
+To return to default values, you can delete a parameter's configuration snippet and restart {microshift-short}.
+//True? If not, what do we do to remove the value/revert back? If we do simply delete a snippet, does the MicroShift default setting apply if there isn't another snippet for the parameter?
```

Comment:
True. Basically this drop-in AND regular (/etc/microshift/config.yaml) configurations are overriding the defaults, so the same principle apply whether you remove a drop-in or the config.
So we have: `defaults` overridden by `config.yaml` overridden by `config.d/00.yaml` overridden by `config.d/10.yaml` and so on.